doc/nrf/releases/release-notes-latest.rst

```diff
@@ -148,6 +148,18 @@ The current |NCS| release is based on Zephyr v2.5.99.
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
+* ARM Architecture:
+
+   * Disallowed FP_HARDABI when building applications with Trusted-Firmware-M
+
+* Boards
+
+   * Enabled building with TF-M for Non-Secure applications on nRF9160 DK
+   * Switched to sda-gpios, scl-gpios DeviceTree properties on nRF9160 DK
+   * Added pwm-led0 alias for nRF5340 Application Core
+   * Enabled TF-M testing in CI on QEMU (mps2_an521_nonsecure)
```

Comment:
Should the fromtree'd changes pulled in to resolve the kconfig loop be mentioned here as well, or do you want to wait until the next upmerge?

Reply:
Not sure. IMHO, the problem with the kconfig loop was never "released" downstream, so we're not really fixing any bug. But I can certainly mention this  in the next upmerge as "cleanup of mps2 board definitions". That's probably better, since we know that all the related patches are in (not only what you picked as from-tree)
Agreed?

Reply:
Sounds good :+1: